spec/factories/price_policies.rb

```diff
@@ -5,13 +5,20 @@
     price_group
     charge_for { InstrumentPricePolicy::CHARGE_FOR[:reservation] }
     association :product, factory: :setup_instrument
-    usage_rate { 10 / 60.0 }
-    usage_subsidy { 0 }
     minimum_cost { 1 }
     can_purchase { true }
     start_date { Time.zone.now.beginning_of_day }
     expire_date { PricePolicy.generate_expire_date(start_date) }
     note { "This is note" }
+    usage_rate { 10 / 60.0 }
+    usage_subsidy { 0 }
+
+    after(:build) do |price_policy, _|
+      if price_policy.daily_booking?
```

Comment:
Should we also make `usage_rate` and `usage_subsidy` be `nil`?

Reply:
I'll move the `usage_rate` and `usage_subsidy` fields specification to an else clause in this block so we allow invalid instances to be build using this factory if desired